docs/user/ppl/functions/json.rst

```diff
@@ -60,3 +60,107 @@ Example::
     | json scalar string  | "abc"                           | "abc"                   |
     | json empty string   |                                 | null                    |
     +---------------------+---------------------------------+-------------------------+
+
+JSON_OBJECT
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_object(key1, value1, key2, value2...)` create a json object string with key value pairs. The key must be string.
+
+Argument type: key1: STRING, value1: ANY, key2: STRING, value2: ANY ...
+
+Return type: STRING
+
+Example::
+
+    > source=json_test | eval test_json = json_object('key', 123.45) | head 1 | fields test_json
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | test_json               |
+    |-------------------------|
+    | {"key":123.45}          |
+    +-------------------------+
+
+JSON_ARRAY
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_array(element1, element2, ...)` create a json array string with elements.
+
+Argument type: element1: ANY, element2: ANY ...
+
+Return type: STRING
+
+Example::
+
+    > source=json_test | eval test_json_array = json_array('key', 123.45) | head 1 | fields test_json_array
+    fetched rows / total rows = 1/1
+    +-------------------------+
+    | test_json_array         |
+    |-------------------------|
+    | ["key",123.45]          |
+    +-------------------------+
+
+JSON_ARRAY_LENGTH
+----------
+
+Description
+>>>>>>>>>>>
+
+Usage: `json_array_length(value)` parse the string to json array and return size, if can't be parsed, return null
```

Comment:
NULL is returned in case of any other valid JSON string, NULL or an invalid JSON.

Reply:
Already update